tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -198,6 +200,13 @@ private static ArgumentParser argParser() {
                 .required(true)
                 .description("either --record-size or --payload-file must be specified but not both.");
 
+        parser.addArgument("--bootstrap-server")
+                .action(store())
+                .required(true)
```

Comment:
@ijuma Done. As a newbie on Kafka source code and policies I'm learning this kind of stuff. Could it be worth then open a KIP for making it required in order to have the `--bootstrap-server` option a really first citizen here ?